daemon/cmd/common.go

```diff
@@ -0,0 +1,42 @@
+// Copyright 2020 Authors of Cilium
```

Comment:
Let's call this file `sockopt.go` or similar. `common.go` is very generic and doesn't tell much about the functionality contained.

Reply:
Done.